.gitignore

```diff
@@ -46,3 +46,4 @@ package-lock.json
 
 # a scratch file for lil scripts/notes
 scratch
+scripts/vaccination_alerts/vaccination-alerts.json
```

Comment:
FWIW- We have a scripts/.gitignore and scripts/alert_emails/.gitignore file.  Not sure what the best practice is for having multiple .gitignore files, but we should maybe consider consolidating into the root .gitignore if we're not a fan (not in this PR). 😁

Reply:
I don't have strong opinions one way or the other, but since we have a `.gitignore` in the scripts directory I'm going to move this there